### ISSUE TITLE
[🐛 Bug]: 

### ISSUE THREAD

**Issue by Djonathan-Kaiper-Carguero** [SeleniumHQ/selenium#11348]
### What happened?

I'm using the version below and it's giving an error when compiling the project

        <dependency>
            <groupId>org.seleniumhq.selenium</groupId>
            <artifactId>selenium-java</artifactId>
            <version>4.5.0</version>
        </dependency>

### How can we reproduce the issue?

```shell
I'm using the version below and it's giving an error when compiling the project


        <dependency>
            <groupId>org.seleniumhq.selenium</groupId>
            <artifactId>selenium-java</artifactId>
            <version>4.5.0</version>
        </dependency>
```

### Relevant log output

```shell
Erro compilação projeto

java: cannot access org.openqa.selenium.Rotatable
  class file for org.openqa.selenium.Rotatable not found
```

### Operating System

MAC

### Selenium version

JAVA 17

### What are the browser(s) and version(s) where you see this issue?

Chorme

### What are the browser driver(s) and version(s) where you see this issue?

108.0

**Comment by Djonathan-Kaiper-Carguero** [comment]
Problem resolution:

Change selenium-java to 4.7.0 and java-client(appium) to 8.3.0

**Comment by titusfortner** [comment]
Yes, `Rotatable` was removed in Selenium 4.7. Appium implemented it in 8.3.
